scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -2074,6 +2078,14 @@ s32 BPF_STRUCT_OPS(layered_init_task, struct task_struct *p,
 	taskc->last_cpu = -1;
 	taskc->layer_id = MAX_LAYERS;
 	taskc->refresh_layer = true;
+	taskc->prev_llc_id = MAX_LAYERS;
+
+	/*
+	 * Start runtime_avg at some arbitrary sane-ish value. If this becomes a
```

Comment:
I guess another alternative if the task were to be assigned a `prev_llc_id` it could just be initialized to the current avg for the `llcc->queued_runtime[layer_id]`, but this should work.

Reply:
`queued_runtime` is the average of the sum of all queued tasks' avg runtimes, which can run up quite high under saturation. Ideally, this I think should be the avg of the first runtimes of the children for each parent.